nixos/doc/manual/installation/installing.xml

```diff
@@ -67,7 +100,25 @@
     <listitem><para>For creating swap partitions:
     <command>mkswap</command>.  Again it’s recommended to assign a
     label to the swap partition: <option>-L
-    <replaceable>label</replaceable></option>.</para></listitem>
+    <replaceable>label</replaceable></option>. For example:
+
+<screen>
+# mkswap -L swap /dev/sda2</screen>
+
+    </para></listitem>
+
+    <listitem>
+    <variablelist>
+      <varlistentry><term>UEFI systems</term>
+      <listitem><para>For creating boot partitions:
+    <command>mkfs.fat</command>.  Again it’s recommended to assign a
+    label to the boot partition: <option>-L
+    <replaceable>label</replaceable></option>. For example:
+
+<screen>
+# mkfs.fat -F 32 -L boot /dev/sda3</screen>
```

Comment:
I'm not sure if it's required, but most people have the EFI boot loader on sda1.

Reply:
The plan behind numbering the partitions the way that I have partly to match the partition numbers in the rest of the installation documentation (sda1 for / and sda2 for /swap), and partly to simplify the merging of the BIOS and UEFI documentation.

I can try something different there, but it could be a bit dicey.

Reply:
I did a small amount of digging, and I don't think the EFI boot loader needs to be on any particular partition.  For some reason I'm seeing sda3 popping up a few times when I google for information on this, although it's only sda3 in my proposed change to limit the scope of the change and try to merge the BIOS and UEFI instructions without it all turning into a mess.